postcss.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = ({file}) => ({
             }),
         require("postcss-nested"),
         require("postcss-extend-rule"),
+        require("postcss-mixins"),
```

Comment:
FYI in case this gets revived, I’ve removed postcss-mixins in #24731, because you can already do this with postcss-extend-rule (as we do for `%dark-theme`).